notebooks/episodic_training.ipynb

```diff
@@ -32,15 +34,53 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
+    "id": "u-qkFjp9h4Mh"
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import google.colab\n",
+    "    colab = True\n",
+    "except:\n",
+    "    colab = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "QTwWMJerh5yF",
+    "outputId": "26dc469a-3b9f-46eb-950e-ce1d322ad126"
+   },
+   "outputs": [],
+   "source": [
+    "if colab is True:\n",
+    "    # Running in Google Colab\n",
+    "    # Clone the repo\n",
+    "    !git clone https://github.com/sicara/easy-few-shot-learning\n",
+    "    %cd easy-few-shot-learning\n",
+    "    !pip install easyfsl\n",
```

Comment:
We cloned the repo and moved at its root dir, so there is already a package `easyfsl`. Installation from PyPi makes it not obvious what version of `easyfsl` we are using. I think it's better if we don't do the pip install and just use the package inside the repo. (Same in other notebook)